frontend/src/services/uploadFile.js

```diff
@@ -12,19 +12,21 @@ import {
 } from '../constants/translationLabels/formElements';
 
 export const uploadStudentsToGroupFile = (file, groupId) => {
+    console.log(file, groupId);
     const formData = new FormData();
     formData.append('file', file);
     axios
         .post(`${STUDENTS_TO_GROUP_FILE}${groupId}`, formData)
         .then((res) => {
+            console.log(res);
             let students = res.data.length;
             students +=
                 students !== 1
                     ? ` ${i18n.t(FORM_STUDENTS_FILE_LABEL)}`
                     : ` ${i18n.t(FORM_STUDENT_FILE_LABEL)}`;
             successHandler(
                 i18n.t(FILE_BACK_END_SUCCESS_OPERATION, {
-                    cardType: students,
+                    cardType: i18n.t(FORM_STUDENTS_FILE_LABEL),
                     actionType: i18n.t(FILE_LABEL),
                 }),
             );
```

Comment:
maybe you should create saga for this request

Reply:
done